README.md

```diff
@@ -73,20 +75,231 @@ Set other options at your discretion. For example:
+        "showReuseMessage": false,
+        "clear": true
+      }
+    },
+    {
+      "label": "optimizt: Optimize Image (lossless)",
+      "type": "shell",
+      "command": "optimizt",
+      "args": [
+        "--lossless",
+        "--verbose",
+        {
+          "value": "${file}",
+          "quoting": "strong"
+        }
+      ],
+      "presentation": {
+        "echo": false,
+        "showReuseMessage": false,
+        "clear": true
+      }
+    },
+    {
+      "label": "optimizt: Create WebP",
+      "type": "shell",
+      "command": "optimizt",
+      "args": [
+        "--webp",
+        "--verbose",
+        {
+          "value": "${file}",
+          "quoting": "strong"
+        }
+      ],
+      "presentation": {
+        "echo": false,
+        "showReuseMessage": false,
+        "clear": true
+      }
+    },
+    {
+      "label": "optimizt: Create WebP (lossless)",
+      "type": "shell",
+      "command": "optimizt",
+      "args": [
+        "--webp",
+        "--lossless",
+        "--verbose",
+        {
+          "value": "${file}",
+          "quoting": "strong"
+        }
+      ],
+      "presentation": {
+        "echo": false,
+        "showReuseMessage": false,
+        "clear": true
+      }
+    }
+  ]
+}
+```
+
+#### How to use
+
+1. Open the file for processing using Optimizt, it should be in the active tab.
+2. Run `>Tasks: Run Task` from the _Command Palette_.
+3. Select the required task.
+
+#### Shortcuts
+
+You can add shortcuts for a specific task by run `>Preferences: Open Keyboard Shortcuts (JSON)` from the _Command Palette_.
+
+An example of adding a hotkey to run the "optimizt: Optimize Image (lossless)" task:
+
+```javascript
+// Place your key bindings in this file to override the defaults
+[
+  {
+    "key": "ctrl+l",
+    "command": "workbench.action.tasks.runTask",
+    "args": "optimizt: Optimize Image (lossless)"
+  }
+]
+```
+
+</details>
+
+### Plugin for Sublime Text 3
+
+<details>
+
+You’ll find the user settings directory in one of the following paths:
+
+- macOS: `~/Library/Application Support/Sublime Text 3/Packages/User`
+- Linux: `~/.config/sublime-text-3/Packages/User`
+- Windows: `%APPDATA%\Sublime Text 3\Packages\User`
+
+#### Add plugin
+
+Inside the settings directory create a file `optimizt.py` with the following content:
+
+```python
+import os
+import sublime
+import sublime_plugin
+
+optimizt = "~/.nodenv/shims/optimizt"
+
+class OptimiztCommand(sublime_plugin.WindowCommand):
+  def run(self, paths=[], options=""):
+    if len(paths) < 1:
+      return
+
+    safe_paths = ["\"" + i + "\"" for i in paths]
+    shell_cmd = optimizt + " " + options + " " + " ".join(safe_paths)
+    cwd = os.path.dirname(paths[0])
+
+    self.window.run_command("exec", {
+      "shell_cmd": shell_cmd,
+      "working_dir": cwd
+    })
+```
+
+Specify path to executable inside `optimizt` variable, this path can be obtained by running `command -v optimizt` in the terminal.
```

Comment:
Does `command` work everywhere? I mean, at least on Windows it probably does not 🤔

Reply:
Good catch 👍
Added information for Windows systems.